packages/components/ng-ovh-utils/src/srefScrollTop/srefScrollTop.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+const moduleName = 'ua.srefScrollTop';
+
+angular
+  .module(moduleName, ['ui.router'])
+  .run(
+    /* @ngInject */ ($transitions, $document, $timeout, $anchorScroll) => {
+      $transitions.onSuccess({}, () => {
+        $document[0].body.style.setProperty('height', 'auto', 'important');
+        $document[0].body.style.setProperty('overflow', 'auto');
+        $anchorScroll();
+        $timeout(function() {
+          $document[0].body.style.removeProperty('height');
+          $document[0].body.style.removeProperty('overflow');
+        });
+      });
```

Comment:
**nitpick:** Depending on transition success event to scroll to top will affect all the screens and not a good idea as there will be some scenarios where the user expected to stay on the current scroll position. So please find a different way to perform the scroll.

Reply:
I had a filter on the transition before but @ghyenne ask me to remove it ;)